source/kernel/kernel.go

```diff
@@ -141,6 +142,12 @@ func (s *kernelSource) Discover() error {
 		s.features.Flags[LoadedModuleFeature] = nfdv1alpha1.NewFlagFeatures(kmods...)
 	}
 
+	if builtinMods, err := getBuiltinModules(); err != nil {
+		klog.Errorf("failed to get builtin kernel modules: %v", err)
+	} else {
+		s.features.Flags[EnabledModuleFeature] = nfdv1alpha1.NewFlagFeatures(builtinMods...)
```

Comment:
I think this feature should combine loadedmodules and builtinmodules, as discussed in #1025